lib/mock/mock-utils.js

```diff
@@ -188,7 +188,11 @@ function buildKey (opts) {
 }
 
 function generateKeyValues (data) {
-  return Object.entries(data).reduce((keyValuePairs, [key, value]) => [...keyValuePairs, key, value], [])
+  return Object.entries(data).reduce((keyValuePairs, [key, value]) => [
+    ...keyValuePairs,
+    Buffer.from(`${key}`),
+    Array.isArray(value) ? value.map(x => Buffer.from(`${x}`)) : Buffer.from(`${value}`)
+  ], [])
```

Comment:
@mcollina

Reply:
This change is fine. It's used for parsing a headers object (key : string) to an array which is what `onHeaders` and `onComplete` expect (which is where this is used).